app/services/npq/is_user_withdrawn.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module NPQ
+  class IsUserWithdrawn
+    attr_reader :user, :cpd_lead_provider
+
+    def initialize(user:, cpd_lead_provider:)
+      @user = user
+      @cpd_lead_provider = cpd_lead_provider
+    end
+
+    def call
+      active_npq_profiles.none?
+    end
+
+  private
+
+    def active_npq_profiles
+      scope = user.npq_profiles
+      scope = scope.includes(npq_application: [:npq_course])
```

Comment:
Do we actually need this line? no `where` clause references this table. I think the `join` at the line below should be sufficient. 
We also don't need this include as we are not going to load them in memory. This query will just generate a "SELECT 1 from ..... WHERE ..."

Reply:
good spot, i must of extracted this from elsewhere that needed it